//let b = 3;

try {

let a = 2 + b;

} catch(e){
    //console.log(e);
    throw new Error(e);
}

console.log('teste');